sunxspex/utils/utils.py

```diff
@@ -1,6 +1,8 @@
 
 import numpy as np
 
+__all__ = ["get_reverse_indices", "_get_reverse_indices"]
```

Comment:
can't the `get_reverse_indices` be deleted as `_get_reverse_indices` is the one used in `thermal_spectrum.py`?

Reply:
Yep!  It could.  I'll do that.